testdata/src/alias/alias.go.golden

```diff
@@ -5,5 +5,5 @@ import "net/http"
 type myHeader = http.Header
 
 func _() {
-	myHeader{}.Get("Tt") // want `non-canonical header "TT", instead use: "Tt"`
+	myHeader{}.Get("Tt") // want `use "Tt" instead of "TT"`
```

Comment:
Is it normal that every golden files contain a valid code but  the comment want reports an error

Here Tt us used in the code, but the want seems unrelated

Reply:
It's okay.

Uses only by tests https://pkg.go.dev/golang.org/x/tools/go/analysis/analysistest#RunWithSuggestedFixes